src/main/java/com/eerussianguy/blazemap/engine/client/LayerRegionTile.java

```diff
@@ -70,6 +70,11 @@ public void save() {
             // FIXME: this needs to hook into a reporting mechanism
             BlazeMap.LOGGER.error("Error saving LayerRegionTile buffer: {}", buffer, e);
         }
+        catch(NullPointerException e) {
+            e.printStackTrace();
+            // FIXME: this needs to hook into a reporting mechanism
+            BlazeMap.LOGGER.error("Error saving LayerRegionTile buffer: {}", buffer, e);
+        }
```

Comment:
Copying the error handling from above, as was getting `NullPointerException`s on world save

Reply:
This is sus, we need to look at this deeper. Looks like BME-38 (TODO in 1.20.x)

Reply:
Indeed, looking at the ticket, it does seem to be the same issue. This should at least minimise client crashing in the mean time.

Reply:
That one is low priority because it didn't cause crashes... has this changed?

Reply:
Can't remember, made that particular change a couple weeks ago. Still, better handled than not?

Reply:
If it doesn't crash let it be. I think it's caught and suppressed further down that's why I didn't do anything.
I'd rather only touch it to really fix it. Which is currently scheduled for 1.20.x

Reply:
Confirmed it doesn't crash when this is removed. Have now removed the additional error handling